interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -43,9 +43,9 @@ public function validcredential($credentials){
 			$pres = $appTable->zQuery("SELECT * FROM patient_access_offsite WHERE portal_username=?",array($credentials[6]));
             $prow = $pres->current();
             
-			$newpatient_to_approve = 0;
+			$patient_encounter_to_approve = 0;
 			if(!$prow['portal_pwd']){
```

Comment:
oops.  I caught some of that...but I guess not all.

Correcting now.


On 2016-09-06 09:30 PM, Terry Hill wrote:
>
> In 
> interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php 
> <https://github.com/LibreEHR/LibreEHR/pull/149#discussion_r77747827>:
>
> > @@ -43,9 +43,9 @@ public function validcredential($credentials){
> >  			$pres = $appTable->zQuery("SELECT * FROM patient_access_offsite WHERE portal_username=?",array($credentials[6]));
> >              $prow = $pres->current();
> >
> > -			$newpatient_to_approve = 0;
> > +			$patient_encounter_to_approve = 0;
> >  			if(!$prow['portal_pwd']){
>
> These are actually new patients to approve from the portal
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/LibreEHR/LibreEHR/pull/149/files/52c73e6f8bd9a59efdc56fcff29497c2982dd941#r77747827>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AAhzF6Q2ecTOljK7QvbOFDPE5VRkxm-Oks5qnhPSgaJpZM4JzTqH>.
>